packages/sdk/src/handlers/contract-kit.ts

```diff
@@ -12,14 +14,21 @@ import { ChainHandler } from './interface';
  */
 export class ContractKitTransactionHandler implements ChainHandler {
   private signedTransaction?: EncodedTransaction;
-  private readonly blockchainAddress;
-  private readonly dekAddress;
+  private readonly blockchainAddress: string;
+  private dekAddress;
 
   constructor(private readonly kit: ContractKit) {
     [this.blockchainAddress, this.dekAddress] = this.kit
       .getWallet()
       .getAccounts();
 
+    (async () => {
+      const accounts = await this.kit.contracts.getAccounts();
+      const res = await accounts.getDataEncryptionKey(this.blockchainAddress);
+      //@ts-ignore
```

Comment:
We should remove this since payments uses eslint. If for some reason you needed it we should look into how ts-lint is bleeding into this project in your setup.